src/core/src/core_logic/PatchInstaller.py

```diff
@@ -435,14 +444,10 @@ def install_packages_in_batches(self, all_packages, all_package_versions, packag
 
             packages_in_batch = []
             package_versions_in_batch = []
-            skip_packages = []
             already_installed_packages = []
 
             for index in range(begin_index, end_index + 1):
-                if package_versions[index] == Constants.UA_ESM_REQUIRED:
```

Comment:
Marker: Check if this constant is stripped out from Constants if not needed anymore in this form (dead code).